RedisLibrary/RedisLibraryKeywords.py

```diff
@@ -582,3 +582,20 @@ def delete_item_from_list_redis(self, redis_conn, list_name, index, item=None):
                 raise AssertionError
         redis_conn.lset(list_name, index, 'DELETE_ITEM')
         redis_conn.lrem(list_name, 1, 'DELETE_ITEM')
+
+    @keyword('Disconnect to Redis')
+    def disconnect_to_redis(self, redis_conn):
```

Comment:
I think the keyword should be "Disconnect from Redis

Reply:
Hi @leonpros, thank you for your feedback. I subimmited the fix with you suggest, can you review again please?